phonenumberutil_test.go

```diff
@@ -605,20 +605,20 @@ func Test_getMetadata(t *testing.T) {
 	}
 }
 
-func Test_isLeadingZeroPossible(t *testing.T) {
-	if !isLeadingZeroPossible(39) {
-		t.Error("Leading 0 should be possible in Italy")
-	}
-	if isLeadingZeroPossible(1) {
-		t.Error("Leading 0 should not be possible in the USA")
-	}
-	if !isLeadingZeroPossible(800) {
-		t.Error("Leading 0 should be possible for International toll free")
-	}
-	if isLeadingZeroPossible(889) {
-		t.Error("Leading 0 should not be possible in non-existent region")
-	}
-}
+// func Test_isLeadingZeroPossible(t *testing.T) {
```

Comment:
If this logic is deprecated, then let's remove this code instead of commenting it out.

Reply:
Done!